src/Mono.Android.Export/CallbackCode.cs

```diff
@@ -511,9 +511,14 @@ static class DynamicCallbackFactory
 	{
 		static DynamicCallbackFactory ()
 		{
-			var ass = AppDomain.CurrentDomain.DefineDynamicAssembly (
+#if MONOANDROID1_0
+			var assembly = AppDomain.CurrentDomain.DefineDynamicAssembly (
 				new AssemblyName ("__callback_factory__"), AssemblyBuilderAccess.Run);
-			Module = ass.DefineDynamicModule ("__callback_factory__");
+#else
+			var assembly = AssemblyBuilder.DefineDynamicAssembly (
```

Comment:
This method exists in the "normal" MonoAndroid profile.  We can skip the `#if MONOANDROID1_0` part and just use this directly.